backend/templates/_listen.conf

```
@@ -5,9 +5,15 @@
   #listen [::]:80;
 {% endif %}
 {% if certificate -%}
-  listen 443 ssl{% if http2_support == 1 or http2_support == true %} http2{% endif %};
-{% if ipv6 -%}
-  listen [::]:443 ssl{% if http2_support == 1 or http2_support == true %} http2{% endif %};
+  listen 443 ssl;
+  {% if http2_support == 1 or http2_support == true %}
+  http2;
+  {% endif %}
+{% if ipv6 -%}/
+  listen [::]:443 ssl;
+  {% if http2_support == 1 or http2_support == true %}
+  http2;
```

The correct statement should be as below:

```suggestion
  http2 on;
```